docs/source/enable/abstract_window.rst

```diff
@@ -0,0 +1,28 @@
+Top-level Windows
+=================
+When a component is shown on screen via a GUI toolkit, its :attr:`window` trait
+contains an instance of :class:`AbstractWindow` which serves as a delegate
```

Comment:
This could link to the api docs

That would also make the following section a little redundant, but I don't think it hurts to call it out explicitly like is done here

Reply:
Done.